rogue_net/rogue_net/relpos_encoding.py

```diff
@@ -14,6 +14,7 @@ class RelposEncodingConfig:
     obs_space: ObsSpace
     d_head: int
     per_entity_values: bool = True
+    exclude_entities: Optional[List[str]] = None
```

Comment:
Can we just make this a `List[str]` with default of `[]` (using dataclasses.field)? (Really it should be `Set[str]` but don't know if that works with our current json loader).